python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -102,18 +103,20 @@ def execute(
         self._start_time = time.perf_counter()
 
         if not isinstance(dag, InputDataBuffer):
-            stdout_logger = logger.get_logger()
-            log_path = logger.get_datasets_log_path()
+            log_path = get_log_path()
             message = "Starting execution of Dataset."
             if log_path is not None:
                 message += f" Full log is in {log_path}"
-            stdout_logger.info(message)
-            stdout_logger.info("Execution plan of Dataset: %s\n", dag)
-            logger.get_logger(log_to_stdout=False).info(
-                "Execution config: %s", self._options
-            )
+            message += f"\nExecution plan of Dataset: {dag}"
+            logger.info(message)
+
+            context = DataContext.get_current()
+            if context.print_on_execution_start:
+                print(message)
```

Comment:
What is this doing? Can this be a log message? I'm wary about including `print` in production.

Reply:
One option is to replace info calls with debug calls except for when we want to actually print a message (like in this case.) I think a lot of our current info logs are better described as debug logs, so I don't think this should be an issue.

@raulchen wdyt?